packages/f-checkout/test/specs/component/f-checkout-delivery.component.spec.js

```diff
@@ -55,4 +55,32 @@ describe('f-checkout "delivery" component tests', () => {
         // Assert
         expect(CheckoutComponent.isFieldErrorDisplayed('mobileNumber')).toBe(false);
     });
+
+    it('should prevent user from submitting a postcode with illegal characters', () => {
```

Comment:
Is there a test to cover the `error-address-postcode-empty` scenerio? 

```javascript
const addressInfo = {
      postcode: ''
};

Reply:
Yep, the forEach test on lines 11 to 18 iterates over the empty postcode error :)

Reply:
Out of interest and not for this PR but those tests look like they're all skipped?

Reply:
Ah yep @kevinrodrigues these are skipped for now because a lot of them were affected by the demo files being removed. We're gonna refactor them for storybook in the next ticket 👍